src/layout/header/Header.tsx

```diff
@@ -18,9 +18,12 @@ import { dataTestId } from '../../utils/dataTestIds';
 import { useFetch } from '../../utils/hooks/useFetch';
 import { CustomerInstitution } from '../../types/customerInstitution.types';
 import { setPartialUser } from '../../redux/userSlice';
+import { MenuButton } from './MenuButton';
 
 export const Header = () => {
   const { t } = useTranslation();
+  const location = useLocation();
+  const currentPath = location.pathname.replace(/\/$/, ''); // Remove trailing slash
```

Comment:
Kunne tatt med en ekstra sikring her også, når jeg tenker meg om
```suggestion
  const currentPath = location.pathname.replace(/\/$/, '').toLowerCase(); // Remove trailing slash
```